Snakefile

```diff
@@ -420,92 +637,1344 @@ rule atlas_experiment_summary:
 	          --output {output.rsummary}
         """
 
-rule copy_raw_gene_counts_from_isl:
+
+
+# rules below are specific for reprocessing
+
+# baseline_rnaseq_experiment
+
+rule add_runs_to_db:
+    """
+    Get run ids from config file and add them to isl db.
+    """
+    conda: "envs/isl-db.yaml"
+    log: "logs/{accession}-add_runs_to_db.log"
+    input:
+        config_xml="{accession}-configuration.xml"
     params:
-        exp_isl_dir=get_isl_dir()
+        db_params=get_db_params()
+    output:
+        temp("logs/{accession}-add_runs_to_db.done")
+    shell:
+        """
+        set -e # snakemake on the cluster doesn't stop on error when --keep-going is set
+        exec &> "{log}"
+        export TNS_ADMIN={params.db_params[0]}/network/admin
+        export LD_LIBRARY_PATH={params.db_params[0]}/lib:$LD_LIBRARY_PATH
+        export PATH={params.db_params[0]}/bin:$PATH
+        export PYTHON_USER={params.db_params[1]}
+        export PYTHON_CONNECT_STRING={params.db_params[2]}
+        export PYTHON_PASSWORD={params.db_params[3]}
+
+        python {workflow.basedir}/isl/db/scripts/get_run_ids_atlas_prod.py {input.config_xml}
+        if [ $? -ne 0 ]; then
+	        echo "ERROR: Failed to parse atlas config file and get run ids for {wildcards.accession} " >&2
+	        exit 1
+        fi
+        touch {output}
+        """
```

Comment:
As discussed today, it depends on whether the cron job deals with this task or not. In the worst case scenario, keeping the rule is harmless:
- it will insert the run_ids collected into LIBRARIES_ATLAS, if they have not been inserted before.
- otherwise does nothing - except some qc checks. For instance if there are no valid run_ids in the experiment, in which case it is also convenient to stop running the analysis. I will also trigger WARN if there are duplicated Run ids or with unexpected accessions.